pkg/build/strategies/sti/sti.go

```diff
@@ -446,13 +446,16 @@ func (b *STI) Execute(command string, config *api.Config) error {
 				}
 				break
 			}
-			if glog.V(2) || config.Quiet != true || command == api.Usage {
-				if (config.UseLogger == true) || glog.V(3) {
-					glog.Info(text)
-					continue
-				}
-				fmt.Fprintf(os.Stdout, "%s\n", strings.TrimSpace(text))
+			// Nothing is printed when the quiet option is set
+			if config.Quiet {
+				continue
+			}
+			// The log level > 3 forces to use glog instead of printing to stdout
+			if glog.V(3) {
+				glog.Info(text)
+				continue
 			}
+			fmt.Fprintf(os.Stdout, "%s\n", strings.TrimSpace(text))
```

Comment:
@soltysh if you have v>=3 then we use glog (with ugly timestamp and prefix), in case you are v<3 we just print the output from assemble to stdout (without timestamps)